packages/backend/src/graphql/resolvers/receiver.ts

```diff
@@ -32,17 +32,23 @@ export const getReceiver: QueryResolvers<ApolloContext>['receiver'] = async (
   return receiverToGraphql(receiver)
 }
 
-export const createReceiver: MutationResolvers<ApolloContext>['createReceiver'] =
+export const createReceiver: MutationResolvers<ForTenantIdContext>['createReceiver'] =
   async (_, args, ctx): Promise<ResolversTypes['CreateReceiverResponse']> => {
     const receiverService = await ctx.container.use('receiverService')
 
+    const tenantId = ctx.forTenantId
+    if (!tenantId) {
+      throw new Error('Tenant id is required to create a receiver')
+    }
```

Comment:
we can just use `ctx.tenant.id` here (since the mutation doesn't take in `tenantId`)